slash/css/src/List/ContentItemDuo/ContentItemDuo.scss

```
@@ -0,0 +1,168 @@
+@use "../../common/common.client.scss" as common;
+
+.af-content-item-duo {
+  display: grid;
+  width: 100%;
+  grid-template:
+    "content-item-duo__icon content-item-duo__label content-item-duo__value"
+    "content-item-duo__icon content-item-duo__label content-item-duo__button";
```

rien contre la nommenclature, au contraire, mais je me dis que vu que ce sont des noms "internes" au `grid` on pourrait peut-être simplifier, cela nous permettrait au passage de grapiller quelques octets de données au chargement du fichier dans le navigateur !

Si tu prends, n'oublies pas de mettre les `grid-area` à jour.

```suggestion
    "icon label value"
    "icon label button";
```

---

Resolved in https://github.com/AxaFrance/design-system/pull/450/commits/94e236e154ff46d3205933d9d3be26cf7e789140